select by tag name/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>JavaScript and the DOM</title>
+  </head>
+  <body>
+    <h1>My List</h1>
+    <ul>
+      <li>This should be red</li>
+      <li>This should be orange</li>
+      <li>This should be yellow</li>
+      <li>This should be green</li>
+      <li>This should be blue</li>
+      <li>This should be indigo</li>
+      <li>This should be violet</li>
+    </ul>
+    <script src="app.js"></script>
```

Comment:
The `app.js` doesn't exist

I think the file should be
```suggestion
    <script src="apps.js"></script>
```